### PR TITLE
chore(#8181): add mobile send message test

### PR DESCRIPTION
**Pull request by kapelo**
# Description

Added E2E test for send message for reports

#8181
https://github.com/medic/care-teams/issues/148

# Code review checklist

- [ ] Readable: Concise, well named, follows the [style guide](https://docs.communityhealthtoolkit.org/contribute/code/style-guide/), documented if necessary.
- [ ] Documented: Configuration and user documentation on [cht-docs](https://github.com/medic/cht-docs/)
- [ ] Tested: Unit and/or e2e where appropriate
- [ ] Internationalised: All user facing text
- [ ] Backwards compatible: Works with existing data and configuration or includes a migration. Any breaking changes documented in the release notes.

# Compose URLs
If Build CI hasn't passed, these may 404:

* [Core](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:e2e-test-mobile-send-message/docker-compose/cht-core.yml)
* [CouchDB Single](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:e2e-test-mobile-send-message/docker-compose/cht-couchdb.yml)
* [CouchDB Cluster](https://staging.dev.medicmobile.org/_couch/builds_4/medic:medic:e2e-test-mobile-send-message/docker-compose/cht-couchdb-clustered.yml)

# License

The software is provided under AGPL-3.0. Contributions to this project are accepted under the same license.

### CODE REVIEW

File: tests/page-objects/default/common/common.wdio.page.js [hunk 1]
```diff
@@ -140,6 +142,14 @@ const closeFastActionList = async () => {
   await (await fastActionListCloseButton()).click();
 };
 
+const isReportActionDisplayed = async () => {
+  await browser.pause(500);
```
Comment (ralfudx) [inline tests/page-objects/default/common/common.wdio.page.js:146]: can this `browser.pause` be avoided?

Comment (kapelo) [reply tests/page-objects/default/common/common.wdio.page.js:146]: For this case no. It fails otherwise.

Comment (kapelo) [reply tests/page-objects/default/common/common.wdio.page.js:146]: Cool, let me try that @garethbowen

File: tests/e2e/default-mobile/reports/messages.wdio-spec.js [hunk 2]
```diff
@@ -0,0 +1,64 @@
+const utils = require('@utils');
+const placeFactory = require('@factories/cht/contacts/place');
+const userFactory = require('@factories/cht/users/users');
+const personFactory = require('@factories/cht/contacts/person');
+const loginPage = require('@page-objects/default/login/login.wdio.page');
+const commonElements = require('@page-objects/default/common/common.wdio.page');
+const reportsPage = require('@page-objects/default/reports/reports.wdio.page');
+const pregnancyFactory = require('@factories/cht/reports/pregnancy');
+
+describe('Report', () => {
```
Comment (latin-panda) [inline tests/e2e/default-mobile/reports/messages.wdio-spec.js:10]:
```suggestion
describe('Report - Send message action', () => {
```
The file can be `send-message.wdio-spec.js` 

I've removed myself as an approver and added @tatilepizs; she's been reorganizing the e2e, so she might have additional feedback

Comment (kapelo) [reply tests/e2e/default-mobile/reports/messages.wdio-spec.js]: Done

**Review by lorerod — APPROVED** [review]
Thank you @kapelo! This looks good to me.

**Review by ralfudx — APPROVED** [review]
looks good

**Comment by tatilepizs** [comment]
Sorry I was too late for this @kapelo, I was trying to wrap up my comments but I started some meetings and forgot to send my review 🫤

**Review by tatilepizs — COMMENTED** [review]
I know that I am late, but I am just going to leave my comments here in case you want to implement them 😊 
Thank you @kapelo

**Comment by kapelo** [comment]
Thanks @tatilepizs , I'll incorporate these changes and open a PR

**Pull request merged**